docker-compose.override.yml

```diff
@@ -5,4 +5,4 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
     ports:
-      - "80"
+      - "80:80"
```

Comment:
Ah, so this was the fix?

Reply:
yes :)
From [Docker Compose Docs](https://docs.docker.com/compose/compose-file/#ports) 

> Either specify both ports (HOST:CONTAINER), or just the container port (an ephemeral host port is chosen).

Before an "ephemeral" port was choosen (could be found in parsing docker info, but it was combersome.
Now http://localhost just works

Reply:
Got it - thank you!